ujson/src/ujson/IndexedValue.scala

```diff
@@ -10,10 +10,12 @@ import upickle.core.{Visitor, ObjVisitor, ArrVisitor, Abort, AbortException}
   * want to work with an AST but still provide source-index error positions if
   * something goes wrong
   */
+@deprecated("Left for binary compatibility, use upickle.core.BufferedValue instead")
```

Comment:
Deprecations should always come with a version, so we can clean them up at some point later.

```suggestion
@deprecated("Left for binary compatibility, use upickle.core.BufferedValue instead", "upickle after 3.1.4")
```